src/cli/commandModules/copy.ts

```diff
@@ -5,24 +5,38 @@ import {
   withStateOption,
   withIgnoreOption,
   withLogLevelOption,
+  withRenameOption,
   ActionArgs,
   CopyArgs,
+  CopyArgsCLI,
   copy,
 } from "../..";
 
 const commandModule: CommandModule<
   ActionArgs & { rootPath: string },
-  CopyArgs
+  CopyArgsCLI
 > = {
   command: "copy <rootPath>",
-  builder: (yargs): Argv<CopyArgs> => {
+  builder: (yargs): Argv<CopyArgsCLI> => {
     return withLogLevelOption(
-      withIgnoreOption(withStateOption(withOutputOption(yargs)))
+      withIgnoreOption(
+        withStateOption(withRenameOption(withOutputOption(yargs)))
+      )
     );
   },
-  handler: async (args: Arguments<CopyArgs>) => {
+  handler: async (args: Arguments<CopyArgsCLI>) => {
     const reporter = new ConsoleActionReporter(args);
-    await copy({ ...args, reporter });
+    let parsed_rename = undefined;
+    // parse rename argument to record if specified
+    if (typeof args.rename === "string") {
+      try {
+        parsed_rename = JSON.parse(args.rename) as Record<string, string>;
+      } catch (SyntaxError) {
+        throw "Unable to parse 'rename' argument. Ensure your 'rename' argument is valid JSON.";
+      }
+    }
+    let args_parsed: CopyArgs = { ...args, rename: parsed_rename };
```

Comment:
This should be a compiler error (let) - I might need to fix configuration. Also please follow the convention of using camelCase:
```suggestion
    const argsParsed: CopyArgs = { ...args, rename: parsed_rename };
```

Reply:
Apologies, I really need to get a JS linter going 😔